### PR TITLE
add step for tagging subnets

### PR DESCRIPTION
**Pull request by geoffcline**
**Description of changes:**
add step to add cluster name tags to subnet resources

```bash
SUBNET_IDS=$(aws cloudformation describe-stacks \
    --stack-name eksctl-${CLUSTER_NAME}-cluster \
    --query 'Stacks[].Outputs[?OutputKey==`SubnetsPrivate`].OutputValue' \
    --output text)

aws ec2 create-tags \
    --resources $(echo $SUBNET_IDS | tr ',' '\n') \
    --tags Key="kubernetes.io/cluster/${CLUSTER_NAME}",Value=
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/getting-started/_index.md [hunk 1]
```diff
@@ -73,6 +73,25 @@ provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication
 necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
 of communicating with identity providers. 
 
+### Tag Subdomains
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:76]: Tag Subnets

File: website/content/en/docs/getting-started/_index.md [hunk 2]
```diff
@@ -73,6 +73,25 @@ provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication
...
+
+Note: If you have a cluster with version 1.18 or below you can skip this step.
+The subnets should be automatically tagged.
+
+```bash
+SUBNET_IDS=$(aws cloudformation describe-stacks \
+    --stack-name eksctl-${CLUSTER_NAME}-cluster \
+    --query 'Stacks[].Outputs[?OutputKey==`SubnetsPrivate`].OutputValue' \
+    --output text)
+
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:89]: I'd drop this newline

File: website/content/en/docs/getting-started/_index.md [hunk 3]
```diff
@@ -73,6 +73,25 @@ provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication
 necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
 of communicating with identity providers. 
 
+### Tag Subdomains
+
+Karpenter expects subnets to be tagged with the cluster name for autodiscovery.
+Retreive the subnet IDs and tag them with the cluster name.
+
+Note: If you have a cluster with version 1.18 or below you can skip this step.
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:81]: We're on 1.21 and 1.18 is officially out of support. I'd leave this out.

File: website/content/en/docs/getting-started/_index.md [hunk 4]
```diff
@@ -73,6 +73,25 @@ provider](https://kubernetes.io/docs/reference/access-authn-authz/authentication
 necessary for IRSA (see below). Kubernetes supports OIDC as a standardized way
 of communicating with identity providers. 
 
+### Tag Subdomains
+
+Karpenter expects subnets to be tagged with the cluster name for autodiscovery.
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:78]: Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.

```...```

File: website/content/en/docs/getting-started/_index.md (hunk 1 above)
Comment (akestner) [inline website/content/en/docs/getting-started/_index.md:76]:
```suggestion
### Tag Subnets
```

File: website/content/en/docs/getting-started/_index.md (hunk 4 above)
Comment (akestner) [inline website/content/en/docs/getting-started/_index.md:78]:
```suggestion
Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.
```

File: website/content/en/docs/getting-started/_index.md (hunk 3 above)
Comment (akestner) [inline website/content/en/docs/getting-started/_index.md:81]:
```suggestion
```

File: website/content/en/docs/getting-started/_index.md (hunk 2 above)
Comment (akestner) [inline website/content/en/docs/getting-started/_index.md:89]:
```suggestion
```

**Review by akestner — COMMENTED** [review]
Echoed @ellistarn's changes as suggestions to make it easy to update

**Comment by geoffcline** [comment]
Thanks! I appreciate the additional info, I was unclear on the impact of this step.

**Review by ellistarn — DISMISSED** [review]

File: website/content/en/docs/getting-started/_index.md [hunk 5]
```diff
@@ -67,6 +67,23 @@ Karpenter will provision EC2 instances in your account.
 
 Additionally, the configuration file sets up [IAM Roles for Service Accounts](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html) (IRSA), which grants Karpenter permissions like launching instances.
 
+### Tag Subnets
+
+Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.
+Retreive the subnet IDs and tag them with the cluster name.
+
+The subnets should be automatically tagged.
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:75]:
```suggestion
```

File: website/content/en/docs/getting-started/_index.md [hunk 6]
```diff
@@ -67,6 +67,22 @@ Karpenter will provision EC2 instances in your account.
 
 Additionally, the configuration file sets up [IAM Roles for Service Accounts](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-enable-IAM.html) (IRSA), which grants Karpenter permissions like launching instances.
 
+### Tag Subnets
+
+Karpenter discovers subnets tagged `kubernetes.io/cluster/$CLUSTER_NAME`. Add this tag to subnets associated configured for your cluster.
+Retreive the subnet IDs and tag them with the cluster name.
+
```
Comment (ellistarn) [inline website/content/en/docs/getting-started/_index.md:74]:
```suggestion
```

**Review by ellistarn — APPROVED** [review]

**Pull request merged**